myia/validate.py

```diff
@@ -102,9 +99,7 @@ def validate_abstract(self, a: (type(None), AbstractExternal), uses):
     P.dot,
     P.switch,
     P.return_,
-    # P.list_map,
-    P.list_append,
-    P.make_list,
+    # P.make_list,
```

Comment:
Stray comment.

Reply:
OK

Reply:
Actually, it's not really a stray comment, unsupported operations are commented out in the whitelist to make it more obvious what is or isn't supported.